UI and usability improvements

foliovision opened the pull request:
Hello,

* we made the lightbox bigger and let it use a monospace font
* we made the select boxes for changed files bigger and resizable
* we made sure the local changes are not thrown away (git reset) when pulling from remote repository - best for websites where some work (such as plugin updates) is done on the live site on the master branch

Thanks,
Martin

Commit:
Removing backup part from pull function
- if database is backed up during pull request, the migration part will import local database backup and not backup pulled from git

fvmartin committed:
Improving the admin UI a bit

fvmartin committed:
Making the staged files list resizable.

fvmartin committed:
Keep local changes when pulling.

fvmartin committed:
Diff aprearance improvement.

mattgrshaw commented:
@foliovision thanks for submitting a PR! Sorry for the delay on this, I'll try to review it this weekend.

Commit:
commit msg: autosuggest commit message

Commit:
auto suggest: improve messages

Commit:
Auto-suggest commit messages for modified plugin files

Commit:
auto-suggest: detect new files in plugin

Commit:
auto-suggest: improve comments

Firiks committed:
Merge branch 'master' into feature-auto-suggest-commit-messages

foliovision committed:
Merge pull request #1 from foliovision/feature-auto-suggest-commit-messages

Feature auto suggest commit messages

Commit:
Improving diff display.

ondics commented:
when do you plan to merge?

fvmartin committed:
Custom SSH key in ~/.ssh/id_github support

fvmartin committed:
PHP 8.2 fix

fvmartin committed:
Avoid command line warnings to be counted as changed files

fvmartin committed:
Deal with git warnings

fvmartin committed:
Revert "Deal with git warnings"

This reverts commit 49de30075e7a0b40ff72e64e78a17e196b009a3d.

fvmartin committed:
WordPress 6.5 fix

fvmartin committed:
Fix PHP 8.2 warning caused by running a filter which expect $post

fvmartin committed:
Do not show admin bar item for admins without install_plugins capability

Using the same permission check as the actual menu item, which does it properly

fvmartin committed:
Revert "Custom SSH key in ~/.ssh/id_github support"

This reverts commit c825cbec303d81d009a7b91b35b606ec2a6f5335.
